aviary/variable_info/enums.py

```diff
@@ -148,36 +123,81 @@ class GASP_Engine_Type(Enum):
 
 
 @unique
-class Flap_Type(Enum):
+class FlapType(Enum):
     """
     Defines the type of flap used on the wing. Used in GASP-based aerodynamics and mass calculations.
     """
-
+    # Plain flaps
     PLAIN = 1
-    """
-    Plain flaps
-    """
+    # Split flaps
     SPLIT = 2
-    """
-    Split flaps
-    """
+    # Single-slotted flaps
     SINGLE_SLOTTED = 3
-    """
-    Single-slotted flaps
-    """
+    # Double-slotted flaps
     DOUBLE_SLOTTED = 4
-    """
-    Double-slotted flaps
-    """
+    # Triple-slotted flaps
     TRIPLE_SLOTTED = 5
+    # Folwer flaps
+    FOWLER = 6
+    # Double-slotted fowler flaps
+    DOUBLE_SLOTTED_FOWLER = 7
+
+
+class LegacyCode(Enum):
     """
-    Triple-slotted flaps
+    Flag for legacy codebases
     """
-    FOWLER = 6
+    FLOPS = 'FLOPS'
+    GASP = 'GASP'
+
+    def __str__(self):
+        return self.value
+
+
+class ProblemType(Enum):
     """
-    Fowler flaps
+    ProblemType is used to switch between different combinations of
+    design variables and constraints.
+
+    SIZING: Varies the design gross weight and actual gross weight to
+    close to design range. This causes the empty weight and the fuel
+    weight to change.
+
+    ALTERNATE: Requires a pre-sized aircraft. It holds the design gross
+    weight and empty weight constant. It then varies the fuel weight
+    and actual gross weight until the range closes to the off-design
+    range.
+
+    FALLOUT: Requires a pre-sized aircraft. It holds the design gross
+    weight and empty weight constant. Using the specified actual
+    gross weight, it will then find the maximum distance the off-design
+    aircraft can fly.
     """
-    DOUBLE_SLOTTED_FOWLER = 7
+    SIZING = 'sizing'
+    ALTERNATE = 'alternate'
+    FALLOUT = 'fallout'
+
+
+class SpeedType(Enum):
+    '''
+    SpeedType is used to specify the type of speed being used.
+    EAS is equivalent airspeed.
+    TAS is true airspeed.
+    MACH is mach
+    '''
+    EAS = auto()
+    TAS = auto()
+    MACH = auto()
+
+
+class Verbosity(Enum):
     """
-    Double-slotted Fowler flaps
+    Sets how much information Aviary outputs when run
+
+    Verbosity levels are based on ubuntu's standard:
+    https://discourse.ubuntu.com/t/cli-verbosity-levels/26973
     """
+    QUIET = 0
+    BRIEF = 1
+    VERBOSE = 2
+    DEBUG = 3
```

Comment:
I like this idea, but we will have to make an issue to propagate this throughout the code.

Reply:
Yep! I'll make an issue for it if this change becomes official